app/models/appointment_summary.rb

```diff
@@ -4,6 +4,10 @@
 
 class AppointmentSummary < ActiveRecord::Base
   DIGITAL_BY_DEFAULT_START_DATE = Date.new(2016, 6, 21)
+  EMAIL_REGEXP = Regexp.union(
+    /\A[^\s][^@]+@([^@\.]+\.)+[^@\.]+[^\s]\z/,
+    /\A\z/
+  )
```

Comment:
As discussed let's just keep this loose for now and keep an eye on the dashboard for failed-to-deliver emails.